arlo-client/package.json

```diff
@@ -62,7 +62,7 @@
     "test:ci": "CI=true tsc && react-scripts test --maxWorkers 2 --env=jsdom --coverage && yarn test:wdio",
     "test:update": "react-scripts test -u  --watchAll=false",
     "test:coverage": "react-scripts test --coverage --watchAll=false",
-    "test:wdio": "pipenv run python ../app.py & pipenv run python ../bgcompute.py & react-scripts start & wdio wdio.conf.js",
+    "test:wdio": "pipenv run python ../app/__init__.py & pipenv run python ../bgcompute.py & react-scripts start & wdio wdio.conf.js",
```

Comment:
I don't love this. WDYT, @benadida?

Reply:
I'm not an expert at python idioms, but my understanding is that one usually wants to have a `.py` file parallel with `./app` that loads the stuff and runs it, so you're never digging into `./app` manually